exchanges/orderbook/linked_list.go

```diff
@@ -526,3 +822,42 @@ func shiftBookmark(tip *Node, bookmark, head **Node, updt Item) bool {
 	(*bookmark).Next = nil
 	return true
 }
+
+// getMovement returns movement on tranche amounts
+func getMovement(leftover, amounts, totalValue, headPrice, refPrice, tranchePrice float64, noLiquidity bool) (*Movement, error) {
```

Comment:
I want to work with you on this soon. But I'm looking at this function to emulate [this](https://en.wikipedia.org/wiki/Slippage_(finance)) example.
Essentially calling:
```go
	movement, err := getMovement(0, 1538.46153846154, 232485.923076923, 151.08, 151.08, 151.25, false)

```
Where 1,538.46153846154 is avg order size, 232,485.923076923 is avg purchase cost, 151.08 is head price and trancheprice (aka final price) is 151.25.
I can see that your averageOrderPrice is correct :tada: but I'm having trouble getting the -717 slippage cost. 

If I wanted to get the slippageCost as per the example, I would need to have the following:
```go
averageOrderPrice := totalValue / amounts
quantityPurchased := 20000
slippageCost := (refPrice * quantityPurchased) - (averageOrderPrice * quantityPurchased)
```
So based on that, this function is missing the total quantity purchased in order to work out this slippage cost. But it seems like you have a different method of calculating is, so that's why I want to work with you to understand what you've done versus the referenced example